files/initramfs-tools/union-mount.j2

```diff
@@ -6,17 +6,17 @@ case $1 in
 esac
 
 ## Mount the aufs file system: rw layer over squashfs
-mkdir -p ${rootmnt}/host/rw
-mount -n -o dirs=${rootmnt}/host/rw:${rootmnt}=ro -t aufs root-aufs ${rootmnt}
+mkdir -p ${rootmnt}/host/image-%%GIT_REVISION%%/rw
+mount -n -o dirs=${rootmnt}/host/image-%%GIT_REVISION%%/rw:${rootmnt}=ro -t aufs root-aufs ${rootmnt}
```

Comment:
Thanks!

---
In reply to: [109391921](https://github.com/Azure/sonic-buildimage/pull/464#discussion_r109391921) [](ancestors = 109391921)